app/assets/javascripts/listings/templates/property-card/_sale-stats-row.html.slim

```diff
@@ -0,0 +1,22 @@
+tr ng-repeat="unit_summary in ::summary" ng-if="::unit_summary.minPriceWithoutParking || unit_summary.minPriceWithoutParking"
+  / Dollar based rent and min income
+  td scope="row"
+    | {{::unit_summary.unitType}}
+  td
+    span
+      span ng-show="::unit_summary.minPriceWithoutParking"
+        | {{::unit_summary.minPriceWithoutParking | currency:"$":0}} or more
+      span ng-show="::!unit_summary.minPriceWithoutParking"
```

Comment:
@jrubenoff the goal here is to only show "or more" if maxPrice > minPrice, right?

Reply:
Yes